src/Stott.Optimizely.RobotsHandler/Stott.Optimizely.RobotsHandler.csproj

```diff
@@ -13,9 +13,9 @@
         <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
         <NeutralLanguage>en-GB</NeutralLanguage>
         <PackageTags>Optimizely CMS Robots Robots_txt</PackageTags>
-        <AssemblyVersion>2.7.0.0</AssemblyVersion>
-        <FileVersion>2.7.0.0</FileVersion>
-        <Version>2.7.0</Version>
+        <AssemblyVersion>2.7.1.0</AssemblyVersion>
```

Comment:
I usually only update these while I package a release as I apply additional values to the `<PackageReleaseNotes>` node which forms part of the nuget package descriptors.